### PR TITLE
Documented the framework.serializer.mapping config option

### LINKED ISSUE #7701: [FrameworkBundle] Allow to configure Serializer mapping paths
See https://github.com/symfony/symfony/pull/21924

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7701.

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1692,6 +1694,19 @@ method.
     For more information, see
     :ref:`component-serializer-handling-circular-references`.
 
+mapping
```
Comment (graillus) [inline reference/configuration/framework.rst]: The target `mapping` is duplicated (already used for the `validation` section)

**Review by graillus — CHANGES_REQUESTED** [review]

File: reference/configuration/framework.rst (hunk 1 above)
Comment (javiereguiluz) [reply reference/configuration/framework.rst]: Thanks! I've fixed it.

File: reference/configuration/framework.rst [hunk 2]
```diff
@@ -1692,6 +1696,21 @@ method.
...
+
+.. _reference-serializer-mapping-paths:
+
+paths
+"""""
+
+**type**: ``array`` **default**: ``[]``
+
+This option allows to define an array of paths with files or directories where
+the component will look for additional validation files.
```
Comment (xabbuh) [inline reference/configuration/framework.rst]: serialization :)

File: reference/configuration/framework.rst [hunk 3]
```diff
@@ -95,7 +95,7 @@ Configuration
     * :ref:`enable_annotations <reference-validation-enable_annotations>`
     * `translation_domain`_
     * `strict_email`_
-    * `mapping`_
+    * :ref:`mapping <reference-validation-mapping>`
```
Comment (yceruto) [inline reference/configuration/framework.rst]: `<reference-serializer-mapping>`

**Comment by weaverryan** [comment]
Typo fixed! Thanks!

**Pull request merged**